「kaminari を使ってページング処理を実装する」の課題を作成しました

a-kuroki-gs opened the pull request.

a-kuroki-gs committed:
「kaminari を使ってページング処理を実装する」の課題を作成しました

maedana commented on app/views/books/index.html.erb:
```diff
@@ -16,3 +16,7 @@
   <%= link_to t('views.common.new', name: Book.model_name.human.downcase), new_book_path %>
 </nav>
+
+<nav class="page-nav">
```
navタグで囲むようにしたのは何か意図があるのですかね?

a-kuroki-gs replied:
こちら、他のリンク(`本の新規作成`や`この本を編集`など)と見た目を合わせたかったため、
`nav`タグをつけております。

maedana commented on app/views/books/index.html.erb:
```diff
@@ -16,3 +16,7 @@
 </nav>
+
+<nav class="page-nav">
+  <%= paginate @books, window: 2 %>
```
念の為確認ですが、windowオプションを指定するとどういう効果あるのか理解して指定してますか?

a-kuroki-gs replied:
`window`オプション：現在のページから見て表示するページ数を指定
(残りが１ページしかなければ、それ以上も表示)

と認識しています。
(課題にも説明を追加しました。)

maedana commented on app/views/kaminari/_first_page.html.erb:
```diff
@@ -0,0 +1,3 @@
+<span class="first">
+  <%= link_to_unless current_page.first?, t('views.pagination.first').html_safe, url, remote: remote %>
+</span>
```
`app/views/kaminari/`配下のファイルは何故必要なのか説明して欲しいです。

a-kuroki-gs replied:
`rails g kaminari:views default`を実行すると見た目を変えることができるとあり実行しましたが、
実際には見た目は特に設定しなかったので、削除しました。

a-kuroki-gs committed:
app/views/kaminari/配下のファイルを削除

rails g kaminari:views defaultを実行すると見た目を変えることができるとあり実行した
が、実際には見た目は特に設定しなかったので、削除
